js/views/teamHistory.js

```diff
@@ -67,13 +67,12 @@ define(["dao", "globals", "ui", "core/player", "lib/bluebird", "lib/jquery", "li
 
                 players = player.filter(players, {
                     attrs: ["pid", "name", "pos", "injury", "tid", "hof", "watch"],
-                    stats: ["gp", "min", "pts", "trb", "ast", "per", "ewa"],
+                    stats: ["season", "abbrev", "gp", "min", "pts", "trb", "ast", "per", "ewa"],
                     tid: inputs.tid
                 });
 
                 for (i = 0; i < players.length; i++) {
                     delete players[i].ratings;
-                    delete players[i].stats;
```

Comment:
After creating players[i].lastYear, still delete players[i].stats. Otherwise it adds a slight overhead by passing it to Knockout.